NuRadioReco/examples/cr_efficiency_analysis/2_II_merge_output_calculate_trigger_rate.py

```diff
@@ -59,16 +58,13 @@
                                                             cfg['number_coincidences'], cfg['coinc_window'])
 
 dic = {}
-for key in cfg:
-    dic[key] = cfg[key]
+dic = cfg
```

Comment:
Just checking: By default, dictionaries will be copied by reference, so if you change dic, you will also change cfg. I don't know if that is a problem here, but if it is, you need to do `dic=cfg.copy()`